pkg/mtbroker/filter/filter_handler_test.go

```diff
@@ -619,3 +620,55 @@ func makeEmptyResponse(status int) *http.Response {
 	r.Header.Set("Content-Length", "0")
 	return r
 }
+
+func Test_filterEventByAttributes(t *testing.T) {
```

Comment:
Just a style matter, could you push the test method before the private methods used for the other tests?

Reply:
It should be `TestFilterEventByAttributes`